README.md

```diff
@@ -1,5 +1,9 @@
 # healthchecks
 
+[![Travis build badge](https://img.shields.io/travis/andela/healthchecks-thunknalla.svg)]()
+[![Coverage Status](https://coveralls.io/repos/github/andela/healthchecks-thunknalla/badge.svg?branch=badge_integration)](https://coveralls.io/github/andela/healthchecks-thunknalla?branch=badge_integration)
```

Comment:
Update the branch to `develop` as stated in the [description](https://www.pivotaltracker.com/story/show/145112661) for this story